discord/types/interactions.py

```diff
@@ -202,6 +203,7 @@ class _InteractionOptional(TypedDict, total=False):
     message: Message
     locale: str
     guild_locale: str
+    app_permissions: Permissions
```

Comment:
Would this be valid? Because technically the type is a `str`

Reply:
I think so

Reply:
tbf idk

Reply:
the property is a Permissions object, the converted argument is a int and the api object is a string